depobs/worker/main.py

```diff
@@ -21,6 +21,15 @@ def scan_npm_package(package_name: str, package_version: str):
     tasks.scan_npm_package_then_build_report_tree(package_name, package_version)
 
 
+@npm_cli.command("advisories")
+@click.argument("package_name", envvar="PACKAGE_USERNAME")
```

Comment:
nit: Ah the command above is wrong the env var should be `PACKAGE_NAME` in both places.